docs/userGuide/contentAuthoring.md

```diff
@@ -125,22 +125,48 @@ You can insert variables to be replaced by pre-defined values anywhere in your s
 
 Some helpful variables are provided by MarkBind for your convenience, as shown below. 
 
+#### Font Awesome Icons
+
+This asset is provided by [Font Awesome](https://fontawesome.com/) under their [free license](https://fontawesome.com/license).
+
+<tip-box>
+Traditional usage:
+<markdown>`<span class='fas fa-chess-knight'></span>`</markdown>
+MarkBind usage:
+<br>
+<code>{<span></span>{fas_chess_knight}}</code>
+</tip-box>
+
+Font Awesome has three different styles for their free icons, each with their own prefix:
+<strong>Solid</strong> (`fas`), <strong>Regular</strong> (`far`) and <strong>Brands</strong> (`fab`).<br>
+Be sure to check Font Awesome's [full list of icons](https://fontawesome.com/icons?d=gallery&m=free) and which styles each icon supports.
+
+Note that you will have to shorten the icon's name before using it within MarkBind. It is a two step process: 

```

Comment:
- "you will *have to* ... before using it" sounds like a chore — and authors don't *have to* use MarkBind's way.
  How about: "MarkBind offers a shorter syntax by transforming the icon's name. It is a two-step process:"

- Use similar wording for Glyphicons.